src/pages/keep-safe/index.js

```diff
@@ -1,156 +1,46 @@
     StyledLink,
-    Card,
     Header,
     Text,
 } from 'components/elements'
 import { Button } from 'components/form'
 import SignupModal from 'components/custom/signup-modal'
 import { localize, WithIntl, Localize } from 'components/localization'
+import Checked from 'images/svg/checklist.svg'
+import Demo from 'images/svg/demo-account.svg'
 
-const GridGrayBackground = styled.div`
-    grid-area: risk;
-    background-color: var(--color-grey-1);
-    padding: 8rem 0;
-
-    section {
-        padding: 0;
-    }
-
-    @media ${device.tabletL} {
-        padding: 4rem 0;
-    }
+const Section = styled(SectionContainer)`
+    background-color: ${props => props.bgcolor || 'transparent'}
 `
-
 const StyledHeader = styled(Header)`
-    max-width: 80rem;
+    max-width: ${props => props.maxwidth || '100%'};
     margin: 0 auto;
     padding: 1.2rem 2rem 4rem;
 `
-const PractiseHeader = styled(Header)`
-    @media ${device.tabletL} {
-        font-size: 4rem;
-    }
-`
-const Practice = styled(Container)`
-    display: grid;
-    grid-template-columns: repeat(2, 1fr);
-    grid-column-gap: 17.4rem;
-    grid-row-gap: 2.4rem;
-
-    div:nth-child(2n) {
-        display: flex;
-        flex-direction: column;
-        align-self: flex-start;
-    }
-    ${Header} {
-        padding-bottom: 1.6rem;
-    }
-    button {
-        padding: 1.6rem 2rem;
-        margin-top: 3.4rem;
-    }
-    @media ${device.laptop} {
-        grid-column-gap: 5rem;
-    }
-    @media ${device.tablet} {
-        grid-template-columns: 1fr;
-    }
-`
-const Risk = styled(FlexGridContainer)`
-    margin-top: 2.7rem;
-
-    ${Text} {
-        font-size: var(--text-size-sm);
-    }
-
-    @media ${device.tabletL} {
-        article {
-            width: 100%;
-            margin: 1.77rem 0 0 0;
-            padding: 3rem 2rem;
-        }
-        article:first-child {
-            margin: 0;
-        }
-    }
-`
-const HeroWrapper = styled.div`
-    @media ${device.tabletL} {
-        div {
-            padding-bottom: 4rem;
-        }
-        h1 {
-            font-size: 6rem;
-        }
-        h4 {
-            font-size: 2rem;
-            line-height: 3.25rem;
-        }
-    }
-`
-const SecureWrapper = styled.div`
-    margin: 0 1.8rem;
-
-    h2 {
-        font-size: 4rem;
-        padding-right: 10rem;
-    }
-    h4 {
-        font-size: 2rem;
-        padding: 1.77rem 0 3rem 0;
-        line-height: 1.5;
-    }
-`
-const KeepSafeSectionContainer = styled(SectionContainer)`
-    @media ${device.tabletL} {
-        padding: 3.55rem 0;
-    }
-`
-const KeepSafeGirdArea = styled.div`
-    @media ${device.tabletL} {
-        display: grid;
-        grid-template-rows: repeat(2, 1fr);
-        grid-template-areas:
-            'risk'
-            'practice';
-    }
-`
-
-const GridSectionContainer = styled(SectionContainer)`
-    grid-area: practice;
-    padding: 8rem 0;
-
-    @media ${device.tabletL} {
-        padding: 4rem 0;
-    }
+const Cta = styled(Button)`
```

Comment:
The normal button styling is not as per Zeplin?

Reply:
No, its larger (padding, border radius)

Reply:
font size and line height can be removed?